app/views/discovered_mailer/discovered_summary.html.erb

```diff
@@ -4,23 +4,22 @@
 <head>
   <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
   <title><%= _('Summary report for discovered hosts from Foreman') %></title>
+  <%= stylesheet_link_tag 'email_summary.css', 'data-turbolinks-track': true %>
```

Comment:
Can you try it without the `.css`?
Do you actually need turbokinks? isn't it an email template?
```erb
<%= stylesheet_link_tag 'email_summary' %>
```

Reply:
Removed them both, and still it is not part of the pipeline :(